templates/elasticsearch-2/0/rancher-compose.yml

```diff
@@ -9,6 +9,12 @@
       type: "string"
       required: true
       default: "es"
+    - variable: "kopf_port"
+      label: "Public Port"
+      description: "Unique public port for kopf"
+      type: "int"
+      default: 80
+      required: false
```

Comment:
If they aren't required, then we would allow blank which would create a syntax error in the compose file because of the colon. I'd set it to required to avoid that.